docs/background.md

```diff
@@ -0,0 +1,95 @@
+# Background
+
+We capture motivational context for the creation of the in-toto Attestation
+Framework here.
+
+## Motivation
+
+This project was initiated with two main motivations:
+
+1.  Support use cases where the prior in-toto link schema was a poor fit. For
+    example, test steps and vulnerability scans are not about "producing" a new
+    artifact so they are awkward to represent in the current format.
+2.  Support interoperability with [Binary Authorization], which will support
+    the agreed-upon format once finalized. This way we have a single ecosystem
+    of software supply chain security.
+
+## Requirements
+
+Functional requirements:
+
+-   Must support user-defined types and schemas, for two reasons:
+    -   To allow in-toto users to more naturally express attestations, as
+        explained above.
+    -   Because Binary Authorization does not want to require its users to use
+        the existing in-toto link schema, which is overly specific.
+-   Should allow indexing of attestations by artifact ID, without having to
```

Comment:
What's an "artifact ID"?

Reply:
I believe this is a subject, or a product in the old Link format.

Reply:
Would it be worth changing 'artifact ID' to digest?

Perhaps a higher level question: can the intro to this doc just clearly indicate this is for historical reference and terms & statements made here were up-to-date at the time it was authored but may be out-of-date now?

Reply:
I already changed to "artifact identifier, i.e., a digest", but can just swap this for digest if preferred?

I've updated the background section to highlight that this is historical reference, good suggestion -- thanks!